packages/@unchainedshop/types/user.ts

```diff
@@ -90,7 +90,7 @@ export type UsersModule = {
   updateInitialPassword: (user: User, initialPassword: boolean) => Promise<void>;
   updateLastBillingAddress: (_id: string, lastAddress: Address, userId: string) => Promise<User>;
   updateLastContact: (_id: string, lastContact: Contact, userId: string) => Promise<User>;
-  updateProfile: (_id: string, profile: UserProfile, userId: string) => Promise<User>;
+  updateProfile: (_id: string, profile: UserProfile & { meta?: any }, userId: string) => Promise<User>;
```

Comment:
Doesn't feel right, can we have updateProfile: (_id: string, { profile?: UserProfile, meta?: any }, userId: string) => Promise<User>

Reply:
Or maybe another "updateMeta"

Reply:
@pozylon why not both? i'll implement both